### PR TITLE
Add contribution rules

### PR DESCRIPTION
**Pull request by ajyong**
This came out of my video call with @johnny-newton.  The link from README.md to CONTRIBUTING.md won't work until this PR has been merged in.

### CODE REVIEW

**Review by johnnewton-hub — APPROVED** [review]
This looks good-thanks for the reviewer role practice. jn

**Review by MarkBennett — CHANGES_REQUESTED** [review]
This is a great start @ajyong.

Can we add a discussion of IP ownership and note that all content must be released to Dev Edmonton Society and will be released under the CC SA 4.0 license.

**Comment by johnnewton-hub** [comment]
Do I need to add my own personal licence.  I'm not worried personally-but it might make me look more legit ;  )

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+We're so glad you're willing to help! There's a few things you should know:
+
+- By contributing to this repository, you are agreeing to release ownership of the intellectual property of the contributions we accept from you to Dev Edmonton Society
```
Comment (MarkBennett) [inline CONTRIBUTING.md:5]: 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Your contributions will be licensed under Creative Commons Attribution-Sharealike 4.0.  See our [license](https://github.com/devedmonton/codevid19.com/blob/master/LICENSE.txt) for more information.
```
Comment (MarkBennett) [inline CONTRIBUTING.md:6]: 👍

**Review by MarkBennett — APPROVED** [review]
Looks good. Ship it! 🚀

**Pull request merged**